tests/integration/conftest.py

```diff
@@ -29,10 +30,33 @@
 logger = logging.getLogger(__name__)
 
 TEST_VARS = dotenv_values("/var/tmp/.env")
+SAT_REG_FILE = dotenv_values("/var/tmp/.env_sat_reg")
+
+
+def satellite_registration_command():
+    """
+    Get the Satellite registration command for the respective system.
+    """
+    sat_reg_command = None
+    if SYSTEM_RELEASE_ENV == "alma-8-latest":
+        sat_reg_command = SAT_REG_FILE["ALMA8_SAT_REG"]
+    elif SYSTEM_RELEASE_ENV == "alma-8.8":
+        sat_reg_command = SAT_REG_FILE["ALMA88_SAT_REG"]
+    elif SYSTEM_RELEASE_ENV == "rocky-8-latest":
+        sat_reg_command = SAT_REG_FILE["ROCKY8_SAT_REG"]
+    elif SYSTEM_RELEASE_ENV == "rocky-8.8":
+        sat_reg_command = SAT_REG_FILE["ROCKY88_SAT_REG"]
+    elif SYSTEM_RELEASE_ENV == "oracle-8-latest":
+        sat_reg_command = SAT_REG_FILE["ORACLE8_SAT_REG"]
+    elif SYSTEM_RELEASE_ENV == "centos-8-latest":
+        sat_reg_command = SAT_REG_FILE["CENTOS8_SAT_REG"]
+    elif SYSTEM_RELEASE_ENV == "oracle-7":
+        sat_reg_command = SAT_REG_FILE["ORACLE7_SAT_REG"]
+    elif SYSTEM_RELEASE_ENV == "centos-7":
+        sat_reg_command = SAT_REG_FILE["CENTOS7_SAT_REG"]
+
+    return sat_reg_command
```

Comment:
If I may suggest some improvements, what do you think of the following code? 

```suggestion
SAT_REG_COMMAND = {
    "alma-8-latest": SAT_REG_FILE["ALMA8_SAT_REG"],
    "alma-8.8": SAT_REG_FILE["ALMA88_SAT_REG"],
    "rocky-8-latest": SAT_REG_FILE["ROCKY8_SAT_REG"],
    "rocky-8.8":SAT_REG_FILE["ROCKY88_SAT_REG"],
    "oracle-8-latest": SAT_REG_FILE["ORACLE8_SAT_REG"],
    "centos-8-latest": SAT_REG_FILE["CENTOS8_SAT_REG"],
    "oracle-7": SAT_REG_FILE["ORACLE7_SAT_REG"],
    "centos-7": SAT_REG_FILE["CENTOS7_SAT_REG"]
}

def satellite_registration_command():
    """
    Get the Satellite registration command for the respective system.
    """
    sat_reg_command = None
    try: 
        sat_reg_command = SAT_REG_COMMAND[SYSTEM_RELEASE_ENV]
    except KeyError:
        print(f"Key not found in satellite registration command dicitonary: {SYSTEM_RELEASE_ENV}")

    return sat_reg_command
```

Reply:
That makes super sense, thank you @r0x0d !